esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4270,7 +4270,8 @@ void CWsWuFileHelper::readWULogToFiles(IConstWorkUnit *cwu, WsWuInfo &winfo, con
     ForEach(*iter)
     {
         const char *processName = iter->query().queryProp("@podName");
-        ILogAccessFilter *processLogFetchFilter = getBinaryLogAccessFilter(logFetchFilter, getPodLogAccessFilter(processName), LOGACCESS_FILTER_and);
+        Owned<ILogAccessFilter> podFilter = getPodLogAccessFilter(processName);
+        ILogAccessFilter *processLogFetchFilter = getBinaryLogAccessFilter(logFetchFilter, podFilter, LOGACCESS_FILTER_and);
```

Comment:
Per our offline conversation, let's be explicit about pointer ownership for the sake of maintainability, this is a low throughput feature and we're ok accepting the added object creation overhead.